packages/crw-preferences/src/encrypted.rs

```diff
@@ -262,7 +262,7 @@ mod test {
         let set_name = "encrypted-exist";
 
         let mut p = EncryptedPreferences::new("password", set_name).unwrap();
-        p.save();
+        p.save().unwrap();
```

Comment:
Wouldn't be better to handle this error here? With `unwrap()` only we can fall in panic-situations of the app without  a clear error message. This will later make debugging harder.